### PR TITLE
Refactor to change switch statement to array

### PR DESCRIPTION
**Pull request by akameco**
What's "異世界"?

### CODE REVIEW

**Comment by nshiba** [comment]
thanks :)
It's great!

**Comment by nshiba** [comment]
"異世界" is new tag.
I forgot to remove this. So please remove "異世界" comment.

File: app/src/main/java/net/nashihara/naroureader/presenter/SearchRecyclerPresenter.java [hunk 1]
```diff
@@ -242,117 +255,48 @@ private void setGenre(Narou narou, List<Integer> genreList) {
...
+        int index = time - 1;
+        int times[][] = {
+          {0, 5},
+          {5, 10},
+          {10, 30},
+          {30, 60},
+          {60, 120},
+          {120, 180},
+          {180, 240},
+          {240, 300}, // 異世界
```
Comment (nshiba) [inline app/src/main/java/net/nashihara/naroureader/presenter/SearchRecyclerPresenter.java]: "異世界" is new tag.
I forgot to remove this. So please remove "異世界" comment.

Comment (akameco) [reply app/src/main/java/net/nashihara/naroureader/presenter/SearchRecyclerPresenter.java]: Done.

**Comment by nshiba** [comment]
thaks!

**Pull request merged**